contracts/CourtRoom.sol

```diff
@@ -98,6 +129,9 @@ contract SwearGame is Owned {
 
 	}
 
+
+
+
 	function deposit(uint256 _depositAmount) onlyOwner payable public returns(bool){
 
 		require(sampleToken.balanceOf(owner) >= _depositAmount);
```

Comment:
do we need this? we should just allow topping up in several rounds.
once the deposit amount is reached the game is launched.